android/app/src/main/java/org/fossasia/openevent/activities/LocationActivity.java

```diff
@@ -101,14 +117,65 @@ protected void onResume() {
         realmRepo.getSessionsByLocation(location)
                 .addChangeListener((sessions, orderedCollectionChangeSet) -> {
                     mSessions.clear();
+                    String upcomingTitle="";
+                    String track="";
+                    String color;
```

Comment:
There's no need to declared and initialise variables, use `String track = session.getTrack.getTitle()`